iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java

```diff
@@ -99,20 +101,35 @@ public Jobs(final IIoTHubWrapper ioTHubService,
 
     @Override
     public CompletionStage<JobServiceModel> scheduleDeviceMethodAsync(
-        String jobId,
-        String queryCondition,
-        MethodParameterServiceModel parameter,
-        Date startTime,
-        long maxExecutionTimeInSeconds)
-        throws ExternalDependencyException {
+            String jobId,
+            String queryCondition,
+            MethodParameterServiceModel parameter,
+            Date startTime,
+            long maxExecutionTimeInSeconds)
+            throws ExternalDependencyException, InvalidInputException {
```

Comment:
nit: Why were the tabs changed?

Reply:
Fixed in https://github.com/Azure/remote-monitoring-services-java/pull/50